app/Http/Controllers/WebController.php

```diff
@@ -67,6 +67,53 @@ public function wordEditSubmit(Request $request, $word){
             'status'=> 'Updated word successfully!',
         ]);
     }
+    public function translationLookup(Request $request, $id){
+        $translationInfo = DB::selectOne("SELECT * FROM `dict_translations` WHERE `id`=?", ["{$id}"]);
```

Comment:
Second argument looks kinda odd, any reason for not just doing "[$id]"?
```suggestion
        $translationInfo = DB::selectOne("SELECT * FROM `dict_translations` WHERE `id`=?", [$id]);
```

Reply:
None. I copied from wordLookup's $translationList and neglected to notice that type cast despite changing the variable name and removing the % wildcards.